tmp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
didn't know you could have .gitignore files in subdirectories! :-D 

Seen people use .gitkeep but [just found out](https://stackoverflow.com/a/7229996) thats not an actual thing more of a convention

Reply:
I noticed it in these repos as I didn't realise that either 😄